otsclient/args.py

```diff
@@ -170,6 +171,17 @@ def parse_ots_args(raw_args):
                               nargs='+',
                               help='Filename')
 
+    parser_stamp.add_argument("--timeout", type=int, default=5,
+                              help="Default maximum timeout for stamp request. "
```

Comment:
We should move this to the stamp subcommand options given it's stamp-specific. Also, a more clear help text would be "Timeout before giving up on a calendar"